specs/accordion.md

```diff
@@ -0,0 +1,184 @@
+
+---
+
+### API
+
+**Accordion**
+*Component Name*
+- `fast-accordion`
+
+*Attributes:*
+- `expandmode` - enum
+  - single
+  - multi - default
+
+*Events*
+- `change: CustomEvent`
+  - no custom data
+  - bubbles
+
+**Accordion Item**
+*Component names:*
+- `fast-accordion-item`
+
+*Attributes:*
+- `expanded` - boolean
+- `id` - string
+- `headinglevel` - enum
+  - 1
+  - 2
+  - 3 - default
+  - 4
+  - 5
+  - 6
+
+*Parts:*
+- panel
+- button
+
+*Slot Names*
+- default
+- heading
+- glyph
+
+### Anatomy and Appearance
+
+```HTML
+<!-- shadow root -->
+<div>
+    <div role="heading" aria-level="3">
+        <button
+            type="button"
+            class="button"
+            part="button"
+            aria-expanded="true"
+            aria-controls="accordion1-panel"
+            id="accordion1"
+        >
+            <slot name="heading" part="heading">Panel one</slot>
+            <slot name="glyph" part="glyph"></slot>
+       </button>
+    </div>
+    <div
+        id="accordion1-panel"
+        role="region"
+        aria-labelledby="accordion1"
+    >
+        Panel one content
+    </div>
+    <div role="heading" aria-level="3">
+        <button
+            type="button"
+            class="button"
+            part="button"
+            aria-expanded="false"
+            aria-controls="accordion2-panel"
+            id="accordion2"
+        >
+            <slot name="heading" part="heading">Panel two</slot>
+            <slot name="glyph" part="glyph"></slot>
+       </button>
+    </div>
+    <div
+        id="accordion2-panel"
+        part="panel"
+        role="region"
+        aria-labelledby="accordion2"
+        class="panel"
+    >
+        Panel two content
+    </div>
+    <div role="heading" aria-level="3">
+        <button
+            type="button"
+            class="button"
+            part="button"
+            aria-expanded="true"
+            aria-controls="accordion3-panel"
+            id="accordion3"
+        >
+            <slot name="heading" part="heading">Panel three</slot>
+            <slot name="glyph" part="glyph"></slot>
+       </button>
+    </div>
+    <div
+        id="accordion3-panel"
+        part="panel"
+        role="region"
+        aria-labelledby="accordion3"
+        class="panel"
+    >
+        Panel three content
+    </div>
+</div>
+
+<!-- end shadow root -->
+```
+
+---
+
+## Implementation
+
+```HTML
+<fast-accordion>
+    <fast-accordion-item>
+        <span slot="heading">Panel one</span>
+        <span slot="glyph">^</span>
+        Panel one content
+    </fast-accordion-item>
+    <fast-accordion-item expanded>
+        <span slot="heading">Panel two</span>
+        <span slot="glyph">^</span>
+        Panel two content
+    </fast-accordion-item>
+    <fast-accordion-item>
+        <span slot="heading">Panel three</span>
+        <span slot="glyph">^</span>
+        Panel three content
+    </fast-accordion-item>
+</fast-accordion>
+```
+
+### States
```

Comment:
It would be great to have a summary of the various states and how one might get into them?